section6/locales-currencies-tz_diff.html

```diff
@@ -0,0 +1,332 @@
+        1. If _ianaTimeZone_ is a Link name, let _ianaTimeZone_ be the corresponding Zone name as specified in the `"backward"` file of the IANA Time Zone Database.
+        1. If _ianaTimeZone_ is `"Etc/UTC"` or `"Etc/GMT"`, return `"UTC"`.
+        1. Return _ianaTimeZone_.
+      </emu-alg>
+
+      <p>
+        The Intl.DateTimeFormat constructor allows this time zone name; if the time zone is not specified, the host environment's current time zone is used. Implementations shall support UTC and the host environment's current time zone (if different from UTC) in formatting.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-defaulttimezone" aoid="DefaultTimeZone">
+      <h1>DefaultTimeZone ()</h1>
+
+      <p>
+        The DefaultTimeZone abstract operation returns a String value representing the valid (<emu-xref href="#sec-isvalidtimezonename"></emu-xref>) and canonicalized (<emu-xref href="#sec-canonicalizetimezonename"></emu-xref>) time zone name for the host environment's current time zone.
+      </p>
+    </emu-clause>
+
+  </emu-clause>
+
+  <ins class="block">
+  <emu-clause id="sec-measurement-unit-identifiers">
+    <h1>Measurement Unit Identifiers</h1>
+
+    <p>
+      The ECMAScript 2019 Internationalization API Specification identifies measurement units using a unit identifier as defined by Unicode Technical Standard #35, Part 2, Section 6. Their canonical form is a string containing all lowercase letters with at least one hyphen.
+    </p>
+
+    <p>
+      All well-formed unit identifiers are allowed. However, the set of combinations of unit identifier and language tag for which localized names and symbols are available is implementation dependent. Where a localized name or symbol is not available, the unit identifier itself is used for formatting.
+    </p>
+
+    <emu-clause id="sec-iswellformedunitidentifier" aoid="IsWellFormedUnitIdentifier">
+      <h1>IsWellFormedUnitIdentifier ( _unitIdentifier_ )</h1>
+
+      <p>
+        The IsWellFormedUnitIdentifier abstract operation verifies that the _unitIdentifier_ argument (which must be a String value) represents a well-formed unit identifier as defined in UTS #35, Part 2, Section 6. In addition to obeying the UTS #35 unit identifier syntax, _unitIdentifier_ must be one of the identifiers anointed in the Validity Data as described in UTS #35, Part 1, Section 3.11; the list may change over time. The following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. If _unitIdentifier_ contains any character that is not in the range `"a"` to `"z"` (U+0061 to U+007A) and is not `"-"` (U+002D), return *false*.
+        1. If _unitIdentifier_ begins or ends with `"-"`, return *false*.
+        1. If _unitIdentifier_ contains zero instances of `"-"`, return *false*.
+        1. If _unitIdentifier_ contains two `"-"` characters directly adjacent to one another, return *false*.
+        1. If _unitIdentifier_ is not in the following list, return *false*.
```

Comment:
Does this step not imply the ones above?

Reply:
See #21.